### ISSUE TITLE
Too many logs

### ISSUE THREAD

**Issue by johncadengo** [rpush/rpush#645]
This is a log I'm getting when I run rpush:

```
18:37:34 rpush_logs.1 |    (0.2ms)  BEGIN
18:37:34 rpush_logs.1 |   ↳ /home/john/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activerecord-5.2.3/lib/active_record/log_subscriber.rb:98
18:37:34 rpush_logs.1 |    (1.0ms)  SELECT  "rpush_notifications"."id" FROM "rpush_notifications" WHERE (processing = FALSE AND delivered = FALSE AND failed = FALSE AND (deliver_after IS NULL OR deliver_after < '2022-09-27 01:37:34.574475')) ORDER BY deliver_after ASC, created_at ASC LIMIT $1 FOR UPDATE  [["LIMIT", 100]]
18:37:34 rpush_logs.1 |   ↳ /home/john/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activerecord-5.2.3/lib/active_record/log_subscriber.rb:98
18:37:34 rpush_logs.1 |    (0.3ms)  COMMIT
18:37:34 rpush_logs.1 |   ↳ /home/john/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activerecord-5.2.3/lib/active_record/log_subscriber.rb:98
```

I'm not sure what's happening at all but it keeps saying this repeatedly until I kill the process.

I didn't change the log level at all. Here is my log level configuration:

```
  config.log_level = (defined?(Rails) && Rails.logger) ? Rails.logger.level : ::Logger::Severity::INFO
```

 Is there a reason why this log is appearing so frequently and should I change the log level to fix it?

**Comment by benlangfeld** [comment]
Which Rails environment is this happening in? What is the Rails log level set to there?

**Comment by johncadengo** [comment]
@benlangfeld this is in development environment. The rails log level is set to `INFO` in my case. Running both `Rails.logger.level` and `::Logger::Severity::INFO` return `1`.

Not sure if this is related, but when I erased the log file `log/rpush.log` it stopped continuing these logs. I'm using foreman to run rpush, which I mentioned in my other issue #644. When I crtl-c foreman, maybe rpush is still running? By removing the file, whatever is printing to that log stops. Then starting up a new instance of rpush seems to work just fine.

**Comment by benlangfeld** [comment]
Can this be closed in favor of #644 then @johncadengo ?

**Comment by johncadengo** [comment]
Sure @benlangfeld. Seems to be related to my workaround and it would be better to not have to do that workaround to get rpush to output properly within foreman. Thanks.
